### PR TITLE
Development

### PR DESCRIPTION
**Pull request by thebinarypenguin**
Our first PR to master

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1 @@
...
-This is a boilerplate project used for starting new projects!
-
-## Set up
-
-Complete the following steps to start a new project (NEW-PROJECT-NAME):
-
-1. Clone this repository to your local machine `git clone BOILERPLATE-URL NEW-PROJECTS-NAME`
-2. `cd` into the cloned repository
-3. Make a fresh start of the git history for this project with `rm -rf .git && git init`
-4. Install the node dependencies `npm install`
```
Comment (thebinarypenguin) [inline README.md:12]: here is a test annotation

**Review by ZolFallows — APPROVED** [review]
i approve this PR

File: src/income/income-router.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+}
+
+
+
+module.exports = incomeRouter
+ 
+/*
+RESTful
+
+example car object
```
Comment (cjcarnivale) [inline src/income/income-router.js:131]: Why do we need a car object?

**Review by cjcarnivale — APPROVED** [review]

File: src/income/income-router.js (hunk 2 above)
Comment (thebinarypenguin) [reply src/income/income-router.js:131]: that was stupid, I'll fix it later

**Review by mdb1710 — APPROVED** [review]
Server looks good to go.

**Pull request merged**